File: src/main/backend/src/main/java/io/server/backend/dto/response/SurveyInfoResponse.java
package io.server.backend.dto.response;

import java.time.LocalDateTime;

public record SurveyInfoResponse(
        Long id,
        String title,
        String description,
        Boolean isRating,
        Boolean isPublic,
        LocalDateTime createdAt,
        SpecialityResponse speciality
) { }

File: src/main/frontend/src/app/models/dto/response/surveyInfoResponse.ts
import {SpecialityResponse} from "./specialityResponse";

export interface SurveyInfoResponse {
  id: number,
  title: string,
  description: string,
  isRating: boolean,
  isPublic: boolean,
  createdAt: string,
  speciality: SpecialityResponse,
  creatorId: number
}

export interface PatientSurveyInfo {
  id: number,
  title: string,
  description: string,
  speciality: string,
  lastPassageDate: string
}
